From 0b9d8b2e52469edb118e9e0dedf8c52f7e6571e1 Mon Sep 17 00:00:00 2001
From: Josh Wilsdon <jwilsdon@joyent.com>
Date: Fri, 9 Jun 2017 11:48:50 -0700
Subject: [PATCH] OS-6171 when using rsync in vmadm we should workaround
 hostile GZ setups Reviewed by: Trent Mick <trentm@gmail.com> Approved by:
 Trent Mick <trentm@gmail.com>

---
 src/vm/node_modules/VM.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/vm/node_modules/VM.js b/src/vm/node_modules/VM.js
index 3ce23319..d4c023f4 100644
--- a/src/vm/node_modules/VM.js
+++ b/src/vm/node_modules/VM.js
@@ -2441,7 +2441,7 @@ function copyFilesystemData(payload, filesystem, log, callback)
 
         // Need to add the trailing '/' in so we copy only the *contents* of the
         // dir instead of the dir itself.
-        args = ['-aq', existing + '/', filesystem.source];
+        args = ['--numeric-ids', '-aq', existing + '/', filesystem.source];
         log.info({
             cmd: cmd,
             args: args,
-- 
2.21.0

